styles/mixins/button.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function () {
+  return function (mixin, bgColor, color) {
```

Comment:
we also need a outline/hollow button. should this be part of this mixin or separate?  I guess we could just create a mixin that calls this mixing with a  border-color and then set bgColor to transparent.  thoughts?